packages/kit/test/prerendering/disabled/svelte.config.js

```diff
@@ -3,7 +3,7 @@ import adapter from '../../../../adapter-static/index.js';
 /** @type {import('@sveltejs/kit').Config} */
 const config = {
 	kit: {
-		adapter: adapter(),
+		adapter: adapter({ fallback: '200.html' }),
```

Comment:
should we rename the app from `disabled` to `fallback`? This test app looks a bit weird as it doesn't check any of the output or anything

Reply:
sure, done